rxv/ssdp.py

```diff
@@ -62,7 +62,10 @@ def discover(timeout=1.5):
 def rxv_details(location):
     """Looks under given UPNP url, and checks if Yamaha amplituner lives there
        returns RxvDetails if yes, None otherwise"""
-    xml = ET.XML(requests.get(location).content)
+    try:
+        xml = ET.XML(requests.get(location).content)
+    except:
```

Comment:
In this specific case you're I'll side with your reasoning. There is only a single line in this try: except: block, almost nothing bad can happend.

But IMO as a general rule it's better to catch even very generic superclass exceptions, I've seen except blocks that silenced ImportErrors or SyntaxErrors in underlying code.